org.eclipse.xtext.testing/src/org/eclipse/xtext/testing/AbstractLanguageServerTest.xtend

```diff
@@ -653,7 +653,7 @@ abstract class AbstractLanguageServerTest implements Endpoint {
 		if (configuration.getAssertSymbols !== null) {
 			configuration.getAssertSymbols.apply(symbols)
 		} else {
```

Comment:
wonder if we should remove
org.eclipse.xtext.testing.AbstractLanguageServerTest.toExpectation(void)
or at least mark it as deprecated+forRemoval